test_stack.py

```diff
@@ -40,7 +40,36 @@ def test_simple_operators():
 
 
 def test_complex_operators():
-    assert eval_program("push 2; push 3; push 5; add; multiply") == [2 * (3 + 5)]
-    assert eval_program("push 36; push 24; push 6; divide; divide;") == [36 / (24 / 6)]
+    assert eval_program("push 2; push 3; push 5; add; multiply") == [
```

Comment:
If the assert is too long and forces you to break lines, perhaps move either side of the condition into a variable.